pkg/utils/environment.go

```diff
@@ -10,8 +10,8 @@ import (
 	"github.com/litmuschaos/chaos-runner/pkg/log"
 )
 
-// GetOsEnv adds the ENV's to EngineDetails
-func (engineDetails *EngineDetails) GetOsEnv() *EngineDetails {
+// SetEnv adds the ENV's to EngineDetails
+func (engineDetails *EngineDetails) SetEnv() *EngineDetails {
```

Comment:
```suggestion
func (engineDetails *EngineDetails) SetEngineDetails() *EngineDetails {
```

fn name seems to be little misleading.